8.x/getting-started/documentation.md

```diff
@@ -2,4 +2,466 @@
 
 [[toc]]
 
-Larammerce is getting documented, as well as developed at high speed. So if you want to be on track, follow the guides.
\ No newline at end of file
+This section guides you to write a document for larammerce project .
+
+## Overall Structure
+
+Each larammerce document should have the parts below :
+
+**1. Introduction :** Describe the overall view of the page , e.g. the subject it is talking about .
+
+**2. Details :** Give technical details about the subject of document .
+
+**3. Sum Up :** Make a brief review of the parts above and write the page references .
+
+## How to write sections and subsections
+
+Each page starts with a `##` header as a section which may continue with one or more `###` headers as subsections .
+
+**EXAMPLE**
+
+ ```
+## Documentation
+
+[[toc]]
+
+This section guides you to write a document for larammerce project .
+
+### Overall Structure
+
+Each larammerce document should have the parts below :
+```
+
+**OUTPUT**
+
+![headers: first image](/06-code.png)
+
+**NOTE :** You can use more than one `##` header in a document .
+
+**NOTE :** A subsection with a `###` header may contain multiple `####` , `#####` or `######` headers .
+
+:::warning Sidebar Index
+Only `##` and `###` headers are indexed in right sidebar as table of contents .
```

Comment:
....
in the right sidebar as the table of contents.
....

Note: Don't put space between the last word and the dot at the of your sentence.